example/flutter_example/lib/main.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:toml/toml.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+      ),
+      home: MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key}) : super(key: key);
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  final tomlParser = TomlParser();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('TOML example'),
+      ),
+      body: FutureBuilder<String>(
+        future: rootBundle.loadString('assets/example.toml'),
```

Comment:
Just a general question: do you think that it would be possible to add something like an `AssetBundleLoader` to the `toml.loader` library such that the user could use `loadConfig` instead of parsing the document manually or wouldn't that work with Flutter's architecture?

Reply:
I'm not familiar with how `loadConfig` works. I think for most cases it would make sense to to add a TOML file as an asset and load it using `rootBundle.loadString`.

Reply:
I read the toml doc now and i think you could make a type of `AssetConfigLoader` which just calles `rootBundle.loadString`